src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/update/OverseasEntityUpdateSubmissionDtoValidatorTest.java

```diff
@@ -0,0 +1,201 @@
+    private OwnersAndOfficersDataBlockValidator ownersAndOfficersDataBlockValidator;
+    private final EntityNameDto entityNameDto = EntityNameMock.getEntityNameDto();
+    private final EntityDto entityDto = EntityMock.getEntityDto();
+    private final PresenterDto presenterDto = PresenterMock.getPresenterDto();
+    private final OverseasEntityDueDiligenceDto overseasEntityDueDiligenceDto = OverseasEntityDueDiligenceMock.getOverseasEntityDueDiligenceDto();
+    private final DueDiligenceDto dueDiligenceDto = DueDiligenceMock.getDueDiligenceDto();
+    private final BeneficialOwnersStatementType beneficialOwnersStatement = BeneficialOwnersStatementType.NONE_IDENTIFIED;
+    private final List<BeneficialOwnerIndividualDto> beneficialOwnerIndividualDtoList = new ArrayList<>();
+    {
+        beneficialOwnerIndividualDtoList.add(BeneficialOwnerAllFieldsMock.getBeneficialOwnerIndividualDto());
+    };
+    private final List<BeneficialOwnerCorporateDto> beneficialOwnerCorporateDtoList = new ArrayList<>();
+    {
+        beneficialOwnerCorporateDtoList.add(BeneficialOwnerAllFieldsMock.getBeneficialOwnerCorporateDto());
+    };
+    private final List<BeneficialOwnerGovernmentOrPublicAuthorityDto> beneficialOwnerGovernmentOrPublicAuthorityDtoList = new ArrayList<>();
+    {
+        beneficialOwnerGovernmentOrPublicAuthorityDtoList.add(BeneficialOwnerAllFieldsMock.getBeneficialOwnerGovernmentOrPublicAuthorityDto());
+    };
+
+    private final List<ManagingOfficerIndividualDto> managingOfficerIndividualDtoList = new ArrayList<>();
+    {
+        managingOfficerIndividualDtoList.add(ManagingOfficerMock.getManagingOfficerIndividualDto());
+    };
+    private final List<ManagingOfficerCorporateDto> managingOfficerCorporateDtoList = new ArrayList<>();
+    {
+        managingOfficerCorporateDtoList.add(ManagingOfficerMock.getManagingOfficerCorporateDto());
+    };
+
+    @Test
+    void testRegistrationSubmissionCalledWhenEntityNumberIsNullAndUpdateFlagTrue() {
+        setIsRoeUpdateEnabledFeatureFlag(true);
+        buildOverseasEntityUpdateSubmissionDtoWithFullDto();
+        Errors errors = overseasEntitySubmissionDtoValidator.validateFull(overseasEntitySubmissionDto, new Errors(), LOGGING_CONTEXT);
+        verify(entityDtoValidator, times(1)).validate(eq(entityDto), any(), any());
+        verify(presenterDtoValidator, times(1)).validate(eq(presenterDto), any(), any());
+        verify(dueDiligenceDataBlockValidator, times(1)).validateDueDiligenceFields(
+                eq(overseasEntitySubmissionDto.getDueDiligence()),
+                eq(overseasEntitySubmissionDto.getOverseasEntityDueDiligence()),
+                any(),
+                any());
+        verify(ownersAndOfficersDataBlockValidator, times(1)).validateOwnersAndOfficersAgainstStatement(eq(overseasEntitySubmissionDto),any(),any());
+        assertFalse(errors.hasErrors());
+    }
+    @Test
+    void testUnsuccessfulOverseasEntityUpdateSubmissionWhenUpdateFlagOff() {
```

Comment:
The name of this method is confusing me - it contains 'Unsuccessful' but the implementation seems to check that there are no errors. Isn't that a 'success' Use Case?